src/packages/@ncigdc/modern_components/SuggestionFacet/SuggestionFacet.js

```diff
@@ -26,7 +25,6 @@ import styled from '@ncigdc/theme/styled';
 import { dropdown } from '@ncigdc/theme/mixins';
 import Link from '@ncigdc/components/Links/Link';
 import CheckCircleOIcon from '@ncigdc/theme/icons/CheckCircleOIcon';
-import { IRawQuery } from '@ncigdc/utils/uri/types';
```

Comment:
This is used in 126 line. 
```suggestion
import { IRawQuery } from '@ncigdc/utils/uri/types';
```